internal/pkg/plugininstaller/goclient/option.go

```diff
@@ -0,0 +1,54 @@
+package goclient
+
+import (
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/devstream-io/devstream/internal/pkg/plugininstaller"
+)
+
+// Options is the struct for parameters used by the goclient install config.
+type Options struct {
+	Namespace              string                  `validate:"required"`
+	StorageClassName       string                  `validate:"required"`
+	PersistentVolumes      []PersistentVolume      `validate:"required"`
+	PersistentVolumeClaims []PersistentVolumeClaim `validate:"required"`
+	Deployment             Deployment              `validate:"required"`
```

Comment:
`required` does not make sense for non-pointer structs. Remove `required` or change to `*Deployment`